data/libraries.ts

```diff
@@ -67,58 +67,84 @@ export const libraries: Array<Library> = [
     componentMatchings: [
       "Auto Complete",
       "Date Picker",
-      "Select",
+      "Cascade Select",
```

Comment:
I see your point but this approach ignores majority of our work. I don't see the component names visible on UI as filters as well so why just not add them as numbers instead of listing by names. Every one has a different take.

Reply:
I can understand your frustration, but note that this tool is here to help people choosing. Going too deep into component listing won't help people. Things to note IMO:
- In the end people will go to your site (ans competitors) and will see the deep work and detailed components, this tool is just a first filter. 
- It may hide some work on other libraries too, not just PrimeVue.
- PrimeVue is ranked 1 in number of components, so this should not be as detrimental.

>  I don't see the component names visible on UI as filters

This may be added in the future, if this tool is used.